src/arch/xtensa/smp/idc.c

```diff
@@ -349,5 +351,5 @@ void idc_free(void)
 			idc_write(IPC_IDCTFC(i), core, idctfc);
 	}
 
-	schedule_task_free(&idc->idc_task);
+	sof_task_free(&idc->idc_task);
```

Comment:
schedule_task_free() - I want all scheduler APIs to start with schedule_

Reply:
Done